src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -68,7 +68,7 @@ public enum Feature {
         SWITCH_PATTERN_MATCHING(),
         @JEP(number=432, title="Record Patterns", status="Second Preview")
         RECORD_PATTERNS,
-        @JEP(number=436, title="Virtual Threads", status="Second Preview")
+        // not used
         VIRTUAL_THREADS,
```

Comment:
Can the enum also constant be removed?

Reply:
Unfortunately not due to the bootstrapping issues with the build. It was the same for sealed classes where the constant couldn't be removed until the end of the release.